### PR TITLE
Added a badge to display current coverity defects

### PR DESCRIPTION
**Pull request by leemgs**
This commit is to append a badge to display current defects of Coverity Scan.

Signed-off-by: Geunsik Lim <geunsik.lim@samsung.com>

---

### CODE REVIEW

**Comment by leemgs** [comment]
* After this PR:

image

File: README.md [hunk 1]
```diff
@@ -1,9 +1,6 @@
 # NNStreamer
 
-[![Gitter][gitter-image]][gitter-url] [![Code Coverage](http://nnsuite.mooo.com/nnstreamer/ci/badge/codecoverage.svg)](http://nnsuite.mooo.com/nnstreamer/ci/gcov_html/index.html) <a href="https://scan.coverity.com/projects/nnsuite-nnstreamer">
-  <img alt="Coverity Scan Build Status"
-       src="https://scan.coverity.com/projects/19225/badge.svg"/>
-</a>
+[![Gitter][gitter-image]][gitter-url] [![Code Coverage](http://nnsuite.mooo.com/nnstreamer/ci/badge/codecoverage.svg)](http://nnsuite.mooo.com/nnstreamer/ci/gcov_html/index.html) <a href="https://scan.coverity.com/projects/nnsuite-nnstreamer"><img alt="Coverity Scan Defect Status" src="https://img.shields.io/endpoint?url=https://nnsuite.mooo.com/nnstreamer/ci/badge/badge_coverity.json" /> </a>
```
Comment (myungjoo) [inline README.md]: For the readability, would it be possible to add a linebreak after each item?

Comment (leemgs) [reply README.md]: Yes. Improved.

**Comment by leemgs** [comment]
To reviewers, 

After applying the changed PR: 

image

**Pull request merged**